src/olympia/addons/models.py

```diff
@@ -1404,77 +1432,6 @@ def get_latest_file(self):
     def in_escalation_queue(self):
         return self.escalationqueue_set.exists()
 
-    def update_names(self, new_names):
```

Comment:
Interesting, it was never used so I assumed that's the way things are somehow. I couldn't find since when it's missing.